### PR TITLE
Burak-uploader

### PR DESCRIPTION
**Pull request by myniqx**
this is how its look like:
image

if you havent upload any image in this session, you can always get images from server ( image ids are saved in local storage and show them when you click this option)
image

in list, you can see a preview and file formats. you can delete image or select a file format to paste code into markdown editor.
image

after you select a file format, it copies image code (image) to clipboard. when set a cursor to markdown editor and paste, it looks like this :
image

### CODE REVIEW

**Comment by MustafaTRKYLMZ** [comment]
It is looking so perfect. But I am wondering that it is possible to upload other format at the same time such as pdf, exel and doc.

**Review by 7alip — APPROVED** [review]
Thanks

File: packages/ui/src/components/FormUploader/MenuImageItem.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  children: ReactNode
+}
+
+const MenuImageButton: FC<MenuImageButtonProps> = ({ url, children }) => {
+  const { onCopy, setValue, hasCopied } = useClipboard('')
+
+  useEffect(() => {
+    // TODO: Change it to link if it's not an image
+    setValue(`![image](${API_URL}${url})`)
+  }, [url])
```
Comment (7alip) [inline packages/ui/src/components/FormUploader/MenuImageItem.tsx:37]: This comment can be removed after fixing

File: packages/ui/src/components/FormUploader/MenuImageItem.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    </Button>
+  )
+}
+
+export const MenuImageItem: FC<MenuImageItemProps> = ({ image, onDelete }) => {
+  const entries = Object.entries(image.formats ?? {})
+
+  return (
+    <Box pos={'relative'}>
+      {/* TODO: Display pdf icon for pdfs, File icon for other file types */}
```
Comment (7alip) [inline packages/ui/src/components/FormUploader/MenuImageItem.tsx:70]: This comment can be removed after fixing.

Icon wrapper box can have the same height as image.

**Review by 7alip — CHANGES_REQUESTED** [review]
Thanks

**Comment by myniqx** [comment]
file icons:
image

it pastes as link when its not an image:
image

**Comment by 7alip** [comment]
@myniqx The only thing left is to remove dimensions part if with or height is not available. It shows only `x` now

**Comment by myniqx** [comment]
> @myniqx The only thing left is to remove dimensions part if with or height is not available. It shows only `x` now

```
           <VStack fontSize={'xs'} spacing={0}>
              <Box>original</Box>
              {isImage ? (
                <Box>
                  {file.width} x {file.height}
                </Box>
              ) : (
                <Box>{(file.size / 1024).toFixed(2)} kb</Box>
              )}
            </VStack>
```
its already removed. if file is image it ll show dimensions otherwise it ll show file size.
and beside image file they dont have fileformats so there ll be only this 'original' box with filesize, u can see in this image : 
image

**Review by 7alip — APPROVED** [review]
Thanks

**Pull request merged**